### DISCUSSION TITLE
A problem on PETSc installation

### DISCUSSION THREAD

**Discussion by xuxiaobei1995** [idaholab/moose#17784] [Q&A Getting Started] [answered]
Hi all,

I'm installing the latest version of MOOSE in an offline way. I downloaded corresponding PETSc with required external packages and executed the update_and_rebuild_petsc.sh but to encounter a problem when PETsc installed STRUMPACK:
![QQ图片20210510173724](https://user-images.githubusercontent.com/34068460/117639340-674a0300-b1b6-11eb-87f0-eb1b5d65184f.png)

Is it necessary for MOOSE to install PETSc with STRUMPACK? If necessary, does anyone know the way to solve this problem? Any suggestion is appreciated.

Xiaobei

**Comment by milljm** [comment, marked as answer]
Strumpack is not necessary. I ran into a similar issue recently, and was told what I am telling you...

**Reply by xuxiaobei1995** [reply]
Thank you Jason! That's really a good news to me.
